src/js/outreach/index.js

```diff
@@ -1,10 +1,13 @@
+import 'js/base/setup';
```

Comment:
This import caused me a lot of headaches.. couldn't figure out why `dayjs().utc` was erroring when fetching a model.  I'm not sure what better way there is to do this.. I guess we could make a local dayjs import that loads the dayjs plugins rather than importing dayjs directly..  but 🤮